docs/script/keyword.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Can you rewrite it with python?
It's a little hard for us to maintain a complex shell script.